docker/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get -qq install -y \
     build-essential \
     perl \
     wget \
-    postgresql \
+    postgresql-9.6 \
```

Comment:
it feels weird that we are doing this here rather than in Dockerfile-synapsepy2 or whatever